base/system/winlogon/winlogon.c

```diff
@@ -376,6 +376,21 @@ RemoveStatusMessage(
     return Session->Gina.Functions.WlxRemoveStatusMessage(Session->Gina.Context);
 }
 
+VOID
+SetLogonTimestamp(
+    _Inout_ PWLSESSION Session)
+{
+    GetSystemTime(&Session->LastLogon);
+}
+
+BOOL
+IsFirstLogon(
+    _In_ PWLSESSION Session)
```

Comment:
These two helper functions could be turned into macros (in the winlogon.h file).

Reply:
OK.